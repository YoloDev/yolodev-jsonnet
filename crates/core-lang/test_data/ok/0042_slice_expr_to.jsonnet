local foo = [0];

foo[:10]
