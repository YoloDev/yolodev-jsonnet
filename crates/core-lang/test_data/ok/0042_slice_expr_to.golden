local foo#2 = [0.0];
std#1["slice"](foo#2, null, 10.0, null)
